src/components/Workspace/Indicator/index.tsx

```diff
@@ -22,35 +29,30 @@ type Props = {
 class WorkspaceIndicator extends React.PureComponent<Props> {
 
   public render(): React.ReactElement {
-    const status = WorkspaceStatus[this.props.status];
-
-    if (status === WorkspaceStatus.STARTING || status === WorkspaceStatus.STOPPING) {
-      return (
-        <span className='workspace-status-indicator'>
-          <span className='workspace-status-spinner ng-scope'>
-            <div className='spinner'>
-              <div className='rect1' /><div className='rect2' /><div className='rect3' />
-            </div>
+    const { status } = this.props;
+
+    let color: ColorType;
+    switch (status) {
+      case WorkspaceStatus[WorkspaceStatus.STOPPED]:
+        color = 'grey';
+        return (<span className={styles.statusIndicator}><StoppedIcon color={color} /></span>);
+      case WorkspaceStatus[WorkspaceStatus.RUNNING]:
+        color = 'green';
+        return (<span className={styles.statusIndicator}><ResourcesFullIcon color={color} /></span>);
+      case WorkspaceStatus[WorkspaceStatus.ERROR]:
+        color = 'red';
+        return (<span className={styles.statusIndicator}><ExclamationCircleIcon color={color} /></span>);
+      case WorkspaceStatus[WorkspaceStatus.PAUSED]:
+        color = 'orange';
+        return (<span className={styles.statusIndicator}><PauseCircleIcon color={color} /></span>);
+      default:
+        color = 'blue';
+        return (
+          <span className={styles.statusIndicator}>
+            <InProgressIcon className={styles.rotate} color="#0e6fe0" />
```

Comment:
Shouldn't it be like that?

```suggestion
            <InProgressIcon className={styles.rotate} color="{color}" />
```

Reply:
ok